server/src/controllers/userController.js

```diff
@@ -0,0 +1,46 @@
+const { User } = require('../models');
+
+exports.postOne = async (req, res) => {
+  try {
+    const { email, firstName, lastName, userName, image } = req.body;
+    const newUser = await User.create({
+      email,
+      firstName,
+      lastName,
+      userName,
+      image,
+    });
```

Comment:
It could be:
`const newUser = await User.create(req.body)`
No need for destructuring and then restructuring it again 😬